components/FormInput/FormInput.module.css

```diff
@@ -0,0 +1,28 @@
+.labelForm {
+    width: 300px;
+    height: 24px;
+
+    display: flex;
+    align-items: center;
+
+    margin-top: 1.5rem;
+    margin-bottom: 2px;
+
+    color: var(--primary-white);
+}
+
+.inputForm {
+    padding: 0.5rem 0.75rem;
```

Comment:
Please use css vars (please check all the file)

Reply:
Reviewed the whole file and changed to css vars. 
Created an extra one for extra small padding.

Reply:
Done